metrics/configs/pr-consistency-config.yaml

```diff
@@ -0,0 +1,89 @@
+metric: pr-consistency
+query: |
+  select /* Calculate the prob a commit flaked on at least one PR job at some point. + prob that a `/test all` command flakes. */
+    timestamp_to_sec(day) day,
+    round(sum(consistent)/count(consistent),4) commit_consistency,
```

Comment:
Is this `88.88%` or `88.8888%`?

Reply:
The values are just `FLOAT`s, there is no concept of a percent type so `88.8888%` is represented as `.88888888` and rounded to `.8889` or `88.89%`